File: app/src/main/java/alexis/tvrs/rickandmortyuniverse/ui/episodes/Episode.java
package alexis.tvrs.rickandmortyuniverse.ui.episodes;

public class Episode {
    int id;
    String name;
    String air_date;
    String episodeOrder;

    public Episode(int id, String name, String air_date, String episobeOrder) {
        this.id = id;
        this.name = name;
        this.air_date = air_date;
        this.episodeOrder = episobeOrder;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getAir_date() {
        return air_date;
    }

    public String getEpisodeOrder() {
        return episodeOrder;
    }
}

File: app/src/main/java/alexis/tvrs/rickandmortyuniverse/wiki/models/webservices/Episode.kt
package alexis.tvrs.rickandmortyuniverse.wiki.models.webservices

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Episode : Serializable {
    @SerializedName("id")
    @Expose
    var id: Int? = null

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("air_date")
    @Expose
    var airDate: String? = null

    @SerializedName("episode")
    @Expose
    var episode: String? = null

    @SerializedName("characters")
    @Expose
    var characters: List<String>? = null

    @SerializedName("url")
    @Expose
    var url: String? = null

    @SerializedName("created")
    @Expose
    var created: String? = null

    companion object {
        private const val serialVersionUID = -5667862611817692642L
    }
}